packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -306,7 +306,12 @@ export default class TradeStore extends BaseStore {
             showUnavailableLocationError(showError, is_logged_in);
         }
 
+        const notification_messages = this.root_store.ui.notification_messages;
+        this.root_store.ui.notification_messages = [];
         const { active_symbols, error } = await WS.authorized.activeSymbols();
+        runInAction(() => {
+            this.root_store.ui.notification_messages = notification_messages;
+        });
```

Comment:
Why are we resetting the `notification_messages`?

Reply:
@balakrishna-binary In order to not show them before the trade page receives active_symbols because the logics in line 316 added before is to wait for active_symbols and then decide whether to show (or not to show when no active_symbols come from BE) the error "Your Gaming account is scheduled to be closed" to UK/IoM clients and other errors to some other clients if applicable. If I remove this reset, pop-up notifications will flicker for such clients before trade page finishes loading when they select DTrade, i.e. before they see the error instead of DTrader. This reset would not be needed if we could rely on some other value instead of active_symbols because now we have to wait for active_symbols query to be sent which is async and causes the error to be shown only after the response is received.

Reply:
@balakrishna-binary I'm still working on refactoring the notifications system. So this solution seems to me to be temporary. Because as a result of refactoring, I'll make the notification toastbox to be shown last. For now i can leave a comment to this part of code, or please tell me your suggestions.

Reply:
@balakrishna-binary I gave it some thought now, and I think I'll remove the fix for the second bug which is a temporary solution we are discussing now. I'll leave only 1 fix for only 1 bug here in a few moments.